src/axom/core/Types.hpp

```diff
@@ -60,8 +58,7 @@ using IndexType = int32;
  * \note The mpi_traits are initialized in Types.cpp
  * \see Types.cpp
  */
-template < class AxomType >
-struct mpi_traits
+template <class AxomType> struct mpi_traits
```

Comment:
This is formatted wierdly. Can this be changed back?

Reply:
I had the same feelings about this. I was looking into it -- I think we just need to set ``AlwaysBreakTemplateDeclarations`` to ``yes``, but, I didn't have a chance to confirm yet.

Reply:
It definitely can, and we can set the option accordingly.  (Most likely the one that @gzagaris mentioned).

Note that this would be a global change, and Axom does not currently always add a line break to template declarations, so it would change other lines of code elsewhere.

Reply:
I think in our codebase it is more common to have a line break after the template declaration

Reply:
I changed the rule. It is now:
```diff
- AlwaysBreakTemplateDeclarations: multiline
+ AlwaysBreakTemplateDeclarations: true
```